hieradata_aws/common.yaml

```diff
@@ -1122,6 +1122,7 @@ govuk_jenkins::jobs::deploy_app_downstream::applications:
   finder-frontend: {}
   government-frontend: {}
   info-frontend: {}
+  hmrc-manuals-api: {}
```

Comment:
This should come before `info-frontend` so we can keep it alphabetical 👍

Reply:
🤦 of course